Sectors/Lines - LACC.txt

```diff
@@ -511,48 +511,65 @@ COORD:N051.29.60.000:E002.00.00.000
 COORD:N051.27.20.000:E002.30.00.000
 
 SECTORLINE:SAS3
-DISPLAY:London AC Clacton:London AC Clacton:EBBU
-DISPLAY:London TC East:London TC East:EBBU
-COORD:N051.38.13.000:E002.29.60.000
+DISPLAY:London AC Clacton:London AC Clacton:EHAA
+DISPLAY:London TC East:London TC East:EHAA
+COORD:N051.38.11.510:E002.29.59.000
 COORD:N051.27.20.000:E002.30.00.000
 
 SECTORLINE:REF1
 DISPLAY:London AC Clacton:London AC Clacton:EHAA
 DISPLAY:London AC Clacton:London AC Clacton:London TC East
-DISPLAY:London TC East:London TC East:London AC Clacton
```

Comment:
Why does TC East care if EHAM is online at this point? All airspace to immediate East belongs to London - not EHAA